src/providers/v3/pool-provider.ts

```diff
@@ -229,6 +229,8 @@ export class V3PoolProvider implements IV3PoolProvider {
       tokenA: token0,
       tokenB: token1,
       fee: feeAmount,
+      initCodeHashManualOverride: undefined,
```

Comment:
what is this for? :o

Reply:
initcode hash on ZkSync is different from other chains https://github.com/Uniswap/sdks/pull/36#discussion_r1628378049, hence we have to explicitly allow for pass in initCodeHashManualOverride for future awareness. In this case, we encapsulated the different initcode hash with v3-sdk.